callstats-java-sdk/src/main/java/io/callstats/sdk/CallStats.java

```diff
@@ -225,5 +242,142 @@ public synchronized void sendCallStatsBridgeStatusUpdateFromQueue() {
 			sendCallStatsBridgeStatusUpdate(bridgeStatusInfo);
 		}
 	}
+
+	public synchronized void sendCallStatsConferenceEvents(CallStatsConferenceEvents eventType, ConferenceInfo conferenceInfo,
+			final CallStatsStartConferenceListener listener) {
+		long epoch = System.currentTimeMillis() / 1000;
+		String apiTS = "" + epoch;
+		String token = getToken();
+		EventInfo event = new EventInfo(eventType.getMessageType(), "{}");
+		if (eventType == CallStatsConferenceEvents.CONFERENCE_SETUP) {
+
+			CallStatsEventMessage eventMessage = new CallStatsEventMessage(CallStatsConst.CS_VERSION, appId, CallStatsConst.END_POINT_TYPE,
+					conferenceInfo.getConfID(), apiTS, token, bridgeId, conferenceInfo.getInitiatorID(), event);
+			String requestMessageString = gson.toJson(eventMessage);
+			httpClient.sendAsyncHttpRequest(CallStatsConst.conferenceEventUrl, CallStatsConst.httpPostMethod, requestMessageString, new CallStatsHttpResponseListener() {
+				public void onResponse(HttpResponse response) {
+					int responseStatus = response.getStatusLine().getStatusCode();
+					if (responseStatus == CallStatsResponseStatus.RESPONSE_STATUS_SUCCESS) {
+						CallStatsEventResponseMessage eventResponseMessage;
+						try {
+							String responseString = EntityUtils.toString(response.getEntity());
+							eventResponseMessage = gson.fromJson(responseString, CallStatsEventResponseMessage.class);
+						} catch (ParseException e) {
+							logger.error("ParseException " + e.getMessage(), e);
```

Comment:
Perhaps we need to improve error logging by adding context of conference that causes crash.